adding a question set for episode 1 (#89)
SanliFaez/OS4Physicists: Assignments/Podcast_reflections/Question_ep1_ethos.md

# Open Science for Physicists

Especially for the course, we have recorded 4 podcast episodes extending the discussions of the course topics. 
Participants are asked to listen to the podcast and answer some related questions. 

## Questions for Episode 1: Ethos of Open Science and the Credibility Cycle

1. What experience Erik mentions as a kickstarter for his praise for open science
	+ Being invited to a citizen science collaboration

> 	+ Extra attention for their ocean streams package after publishing it online with an open license

Reviewer: ```suggestion
	+ Extra attention for their ocean particle tracking package after publishing it online with an open license
```

	+ The opprotunity of reaching audiences outside academia 
2. Where did Bruno Latour perform his research for the book "Laboratory Life"
	+ Salk Institute, San Diege
	+ Institute of Advanced Studies, Princeton
	+ Institute Pierre and Marie Curie, Paris
3. Which Mertonian norm for Science does Sanli specify in the podcast?
	+ Communism
	+ Universalism
	+ Disinterestedness
	+ Organized skepticism
4. Which example is *not* mentioned by Sanli as a negative side effect of focusing on personal pursuit of credibility
	+ Hypercompetition
	+ Avoiding multi-disciplinary work
	+ Avoiding team collaborations
	+ Not publishing negative results

> 5. Which of the following is not included in the credibility cycle describe by Latour and Woolgar? 

Reviewer: ```suggestion
5. Which of the following is not included in the credibility cycle described by Latour and Woolgar? 
```

	+ Money
	+ Data
	+ Societal relevance
	+ Equipment
	+ Publications

> 6. Which school classification of open science does Sanli name first?

Reviewer: Ay, this would be a very difficult question, if students have to answer this after they listened. Or will they see the questions before listening?

	+ Democratic school 
	+ Infrastructure school
	+ Measurement school 
	+ Public school 
	+ Pragmatic school

> 7. Which argument, according to Sanli, shows that even master projects must consider the increasing complexity of science?

Reviewer: ```suggestion
7. Which argument, according to Sanli, shows that even MSc thesis projects must consider the increasing complexity of science?
```

> 	+ Science integrity applies to all researchers, even undergraduate students

Reviewer: ```suggestion
	+ Science integrity applies to all researchers, even (under)graduate students
```

> 	+ The amount of scientific publications produced in recent years is much more than the last decades

Reviewer: ```suggestion
	+ The amount of scientific publications produced in recent years is much more than in the last decades before
```

	+ Concepts from different disciplines have become relevant because of interdisciplinary research
	+ Large language models have made it easy to fake reports
8. How does Sanli summarize the finding of Latour and Woolgar?

> 	+ Scientists need personal credibility to get into the upper league of science
> 	+ The scientific endeavor cannot be separated from the scientist as a person
> 	+ Science is not mainly a pursuit of objective facts but a race for more resources and recognition

Reviewer: Better to phrase in past tense?
```suggestion
	+ Scientists needed personal credibility to get into the upper league of science
	+ The scientific endeavor couldn't be separated from the scientist as a person
	+ Science was not mainly a pursuit of objective facts but a race for more resources and recognition
```
